packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import { switchToBlockType } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { displayShortcut } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import { Group, UnGroup } from './icons';
+
+export function ConvertToGroupButton( {
+	onConvertToGroup,
+	onConvertFromGroup,
+	isGroupable = false,
+	isUnGroupable = false,
+} ) {
+	return (
+		<Fragment>
+			{ isGroupable && (
+				<MenuItem
+					className="editor-block-settings-menu__control block-editor-block-settings-menu__control"
+					icon={ Group }
+					onClick={ onConvertToGroup }
+					shortcut={ displayShortcut.primaryAlt( 'g' ) }
+				>
+					{ _x( 'Group', 'verb' ) }
```

Comment:
Unless my eyes are deceiving me, I don't think these menu items (at least the icon) are aligned or colored the same as their siblings.

![image](https://user-images.githubusercontent.com/1779930/58832355-c2fbd100-861c-11e9-8743-0b541738c231.png)

Reply:
Great spot. Thinking about this I believe @kjellr mentioned on a previous review but it got lost. 

I've amended to ensure the SVG is 20x20 and all the `fill` attrs are removed. This normalises the color.

## Group
![Screen Shot 2019-06-04 at 10 40 38](https://user-images.githubusercontent.com/444434/58869062-6ac4dd80-86b5-11e9-8933-755d40b8e3ec.png)

## Ungroup
![Screen Shot 2019-06-04 at 10 40 59](https://user-images.githubusercontent.com/444434/58869064-6bf60a80-86b5-11e9-8ad2-f79c4e46f41d.png)

@kjellr Are you happy with these as they stand just to get this merged? I know you have a separate PR to update these pending, but I'd still like to defer that to avoid blocking this PR if that's ok?

Reply:
Thanks, @getdave, that's good for now. I'll follow up with the new icons shortly after this gets merged in.